### PR TITLE
date shortcodes

### PR DESCRIPTION
**Pull request by ChristofLee**
Add shortcodes to display:
- Day
- Month
- Year
- Current date in user defined format

### CODE REVIEW

**Comment by ChristofLee** [comment]
Screenshot 2021-10-15 at 09 13 20

**Comment by ChristofLee** [comment]
Screenshot 2021-10-15 at 09 25 44

File: shortcodes-wp.php [hunk 1]
```diff
@@ -54,3 +54,18 @@ function current_month() {
...
+ * – Else fallback to the default option.
+ */
+add_shortcode('wpinfo_current_date', 'current_date');
+function current_date($atts) {
+    $default_date_format = get_option( 'date_format' );
+    $atts = shortcode_atts( array(
+        'format' => $default_date_format,
+    ), $atts );
+    return get_the_date( $atts['format'] );
+}
```
Comment (warengonzaga) [inline shortcodes-wp.php]: I would like to know how does the user can customize the format? Do you have a clear usage for this as a shortcode? @ChristofLee

**Comment by ChristofLee** [comment]
Is this ok? are you waiting on me here or are you happy to accept?

**Comment by warengonzaga** [comment]
@ChristofLee yes, I'm waiting for a response about how to use the "user-defined date" shortcode?

File: shortcodes-wp.php (hunk 1 above)
Comment (warengonzaga) [reply shortcodes-wp.php]: Pay attention to this @ChristofLee, I'm curious how to use this.

**Comment by ChristofLee** [comment]
I see, apologies I must have missed your message and I couldn't find it looking back.

The user defined date format is what the user can set on the Settings page of the WordPress website.

If a user updates the format on the settings page to something like 'd/m/Y', then all instances of our shortcode will update to show this format too.

However if a user sets their own format in the shortcode, like 'm/d/Y', then it will always display this format.

I hope that's clear but please let me know if you have any more questions or concerns.

**Comment by warengonzaga** [comment]
> I see, apologies I must have missed your message and I couldn't find it looking back.
> 
> The user defined date format is what the user can set on the Settings page of the WordPress website.
> 
> If a user updates the format on the settings page to something like 'd/m/Y', then all instances of our shortcode will update to show this format too.
> 
> However if a user sets their own format in the shortcode, like 'm/d/Y', then it will always display this format.
> 
> I hope that's clear but please let me know if you have any more questions or concerns.

Do you have an example of the usage so I can document it before I merge the update?

Also this PR has conflicts kindly check it.

**Comment by ChristofLee** [comment]
I understand, I will sort this for you asap.

**Comment by ChristofLee** [comment]
Good morning, I have fixed the conflicts and added documentation.

Please let me know if you have any concerns.

**Review by warengonzaga — APPROVED** [review]
This is now approved, we can improve this on the dev branch. I like your enthusiasm for the project. Cheers!

**Pull request merged**